lib/assets/javascripts/mail.js

```diff
@@ -0,0 +1,377 @@
+//= require i18n
```

Comment:
This seems a bit out of place in this PR?

Reply:
Whoopsie, this was left from the merge hell. This file is already present somewhere else